### PR TITLE
feat: add opBNB Mainnet

### PR DESCRIPTION
**Pull request by JunayedMh**
### Description
I did find all the data on opbnb explorer and bnb docs. Also i updated the logo running through https://jakearchibald.github.io/svgomg/

### Backward compatibility

 No

### Testing
Yes

### CODE REVIEW

File: .changeset/ninety-papayas-heal.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@hyperlane-xyz/registry": patch
```
Comment (jmrossy) [inline .changeset/ninety-papayas-heal.md:2]:
```suggestion
"@hyperlane-xyz/registry": minor
```

File: opbnb/addresses.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+domainRoutingIsmFactory: "0x1AA7e1328BC28f585853A8d62F4cF0eB43ABAe8c"
```
Comment (jmrossy) [inline opbnb/addresses.yaml:1]: You new files should be under the `chains` folder

Comment (JunayedMh) [reply opbnb/addresses.yaml]: done,

**Comment by JunayedMh** [comment]
all done.

**Comment by JunayedMh** [comment]
In the last pr i didn’t add the u2utestnet on the chains folder, so i changed the location to chains now of the u2utestnet folder..please kindly merge it.

**Review by jmrossy — COMMENTED** [review]
You additions look good but your files aren't quite formatted correctly. That's why the `prettier` CI test is failing. Can you please run `yarn prettier` to have it format the files?

**Comment by JunayedMh** [comment]
made the files prettier using the command,kindly check again.

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**